runtime/layers/fpm-dev/Dockerfile

```diff
@@ -4,6 +4,20 @@ FROM $LAYER_IMAGE
 # Override the config so that PHP-FPM listens on port 9000
 COPY php-fpm.conf /opt/bref/etc/php-fpm.conf
 
+USER root
+ENV current_os=linux
```

Comment:
Is there a reason we need to change those 2 settings?

Reply:
I would rather remove this if this isn't necessary (keep things as simple as possible).

Reply:
The root user was necessary cause the lambci image which was used for the fpm image uses a different user and you can‘t write to /opt when not root

Reply:
And the current os variable is only used for the download url of blackfire

Reply:
OK I see now! I think I have an idea with multi-stage builds. I'll try that after #490